teams/compiler-contributors.toml

```
@@ -0,0 +1,27 @@
+name = "compiler-contributors"
```

Does it make sense for this to be a subteam of the compiler team?
```suggestion
name = "compiler-contributors"
subteam-of = "compiler"
```

---

I don't really know what "subteam" means, actually

---

In the GH sense, it is not a subteam.

---

I think that on the website, if the compiler contributors were not a subteam, then the two teams would be listed as siblings on the [teams page](https://www.rust-lang.org/governance). But if added as a subteam, then the compiler contributors would be shown on the same page as the compiler team. 

For example, the [Cargo team is a sub-team of the Dev Tools team](https://github.com/rust-lang/team/blob/65f39551f40ed67b0d305c858637aef3f72a78ca/teams/cargo.toml#L1-L2) and appears on [the Dev Tools team page](https://www.rust-lang.org/governance/teams/dev-tools).

---

I don't know how this would affect the GitHub team structure.

---

This doesn't affect the github teams at all.

---

Yeah, subteam is only used as part of the website.